frontend/src/Teacher/InbusImport.vue

```diff
@@ -237,7 +247,18 @@ function onInbusSubjectSelected(event: Event) {
         <td>{{ ca.educationTypeAbbrev }}/{{ ca.order }}, {{ ca.subjectVersionCompleteCode }}</td>
 
         <td>
-          {{ ca.teacherFullNames }}
+          <span v-if="ca.teacherFullNames">{{ ca.teacherFullNames }}</span>
+          <span v-else>
+            <select @change="onTeacherSelected">
+              <option
+                v-for="teacher in teachers"
+                :key="teacher.username"
+                :value="[ca.concreteActivityId, teacher.username]"
```

Comment:
We should just format it, rather than depending on the serialization of an array. Something like this:
```typescript
:value="`${ca.concreteActivityId},${teacher.username}`"
```

Reply:
Thanks, will force push a fix.